### PR TITLE
Fix file upload

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
More than one file cannot be uploaded in task details page

### Solution:
Changes done in upload-file component

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
go task details page and upload files

### CODE REVIEW

**Review by Pendyala-thanvi — APPROVED** [review]

File: src/app/body/task-details/upload-files/upload-files.component.html [hunk 1]
```diff
@@ -12,13 +12,24 @@
...
     <div class="row" style="max-height: 40px;">
         <h4 class="col-md-10 col-10 pt-2 mb-0">Uploaded Files</h4>
         <div class="col-md-1 col-2 pt-1">
             <form>
                 <div class="form-group">
-                    <span class="material-icons uploadIcon">cloud_upload</span>
+                    <label role="button" id="exampleFormControlFile1">
+                    <span class="material-icons uploadIcon"  id="exampleFormControlFile1">cloud_upload</span>
                   <input type="file" style="display: none;" class="form-control-file" id="exampleFormControlFile1" (change)=detectFiles($event)>
+                </label>
```
Comment (viv9k) [inline src/app/body/task-details/upload-files/upload-files.component.html:32]: the ui piece of code already there, we just need to check why it's not working

**Review by viv9k — CHANGES_REQUESTED** [review]
Please understand and do it again

File: src/app/body/task-details/upload-files/upload-files.component.html (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/task-details/upload-files/upload-files.component.html]: Done sir

**Review by sanjaykrishna1203 — APPROVED** [review]
Approved

**Review by viv9k — COMMENTED** [review]
I need demo on this

File: src/app/body/task-details/upload-files/upload-files.component.html [hunk 2]
```diff
@@ -13,12 +13,16 @@
 ***********************************************************/ -->
 <div *ngIf="(uploadService.filesData )?.length != 0; else upload">
     <div class="row" style="max-height: 40px;">
-        <h4 class="col-md-10 col-10 pt-2 mb-0">Uploaded Files</h4>
+        <h4 class="col-sm-10 col-10 pt-2 mb-0 mr-4">Uploaded Files</h4>
```
Comment (viv9k) [inline src/app/body/task-details/upload-files/upload-files.component.html:16]: why using sm, not a good bootstrap that we follow, also mr is not good strategy, try something else

**Review by viv9k — CHANGES_REQUESTED** [review]
Need changes

**Pull request closed without merging**